### PR TITLE
Feature issue remove rename sheet

### LINKED ISSUE #590: 为什么没有删除sheet 的方法？(Why is there no way to delete a sheet?)

### PR DESCRIPTION
**Pull request by rentiansheng**
### feature 

- remove a sheet from file #590
- rename a sheet from file  #602

close #590 #602

### CODE REVIEW

File: file.go [hunk 1]
```diff
@@ -436,3 +436,36 @@ func (f *File) ToSliceUnmerged() (output [][][]string, err error) {
...
+			continue
+		}
+		newSheets = append(newSheets, sheet)
+	}
+	f.Sheets = newSheets
+	delete(f.Sheet, sheetName)
+}
+
+// RenameSheet rename  Sheet
+func (f *File) RenameSheet(oleSheetName, newSheetName string) {
```
Comment (Doctor-X) [inline file.go:455]: In RenameSheet(), I can't see where sheet is getting initialized

Comment (tealeg) [inline file.go:455]: I believe `oleSheetName` should be `oldSheetName`.

File: file.go [hunk 2]
```diff
@@ -436,3 +436,36 @@ func (f *File) ToSliceUnmerged() (output [][][]string, err error) {
...
+	// wait change name sheet
+	var sheet *Sheet
+	for _, sheetItem := range f.Sheets {
+		if sheetItem.Name == oleSheetName {
+			sheetItem.Name = newSheetName
+			break
+		}
+	}
+
+	if sheet != nil {
```
Comment (tealeg) [inline file.go:466]: This block will never run, as `sheet` is never set.

**Review by tealeg — CHANGES_REQUESTED** [review]
It looks good in principle, there's a couple of little tidies needed.

File: file_test.go [hunk 3]
```diff
@@ -1082,3 +1082,36 @@ func (s *SliceReaderSuite) TestFileWithEmptyCols(c *C) {
...
+	c.Assert(f.Sheet["RemoveSheet"], Equals, sheet)
+
+	f.RemoveSheet("RemoveSheet")
+	c.Assert(len(f.Sheets), Equals, 0)
+	var nilSheet *Sheet
+	c.Assert(f.Sheet["RenameSheet1"], Equals, nilSheet)
+}
+
+// Test that we can remove a sheet to a File
+func (l *FileSuite) TesRenameheet(c *C) {
```
Comment (Doctor-X) [inline file_test.go:1104]: This function will never be executed, should be named TestRenameSheet()

**Review by tealeg — APPROVED** [review]

**Pull request merged**